### ISSUE TITLE
CV to COPO integration

### ISSUE THREAD

**Issue by shaw2thefloor** [CyVerseMetadataHack/HackathonIdeas#2]
Obviously the inverse operation could work as well. When someone has finished an analysis in CV, the data could be pushed to COPO and then deposited into repos. We have an embryonic REST api to allow some of this functionality, although careful consideration would need to be given to make sure objects coming from CV to COPO were correctly annotated. For this reason, I think the idea of pushing from COPO to CV is probably easier.

**Comment by ramonawalls** [comment]
If you give us guidance on what sort of annotation is needed, we can work on getting it added. If I understand what you want to do, it should be feasible with the CyVerse metadata API.

**Comment by shaw2thefloor** [comment]
So like I said in another post, I am currently implementing support for Dublin Core datasets. Since thats a very straight forward schema, maybe we should start there. The other schemas we currently have implemented are Figshare and ENA. ENA might be a useful endpoint for your users perhaps?

**Comment by ramonawalls** [comment]
Yes. This makes more sense now. I think, however, we may first need to have a discussion about publishing capabilities -- comparing CV and COPO -- and future plans.

**Comment by niravmerchant** [comment]
It would be good to review the "publishing" framework concept (essentially a dedicated app) in DE that targets various "publishers" but renders specific UI for that end point (and prefills known metadata). A session with @dennisroberts71 and @michael-conway will help align some ideas. This is similar to NCBI SRA and WGS pipeline submission, but more generalized.

**Comment by ramonawalls** [comment]
Heads up @shaw2thefloor - A number of people want to listen in on the COPO presentation, which is scheduled for 10:30 on Thursday.
